packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -81,6 +84,12 @@ const MILLI = 1_000_000n;
  *     governorCreatorFacet: GovernedContractFacetAccess<VaultFactoryPublicFacet, VaultFactoryCreatorFacet>,
  *     adminFacet: AdminFacet,
  *   },
+ *   auctionKit: {
```

Comment:
auction<em>eer</em> throughout to match contract name

Reply:
I'm not sure what consistency you're looking for. I see `stakeFactoryKit`, `PSMKit`, `ammKit`, `feeDistributorKit`, `econCharterKit`, and `vaultFactoryKit`..

Reply:
most of those are `${contractName}Kit`. 
```suggestion
 *   auctioneerKit: {
```

Reply:
Now that the AMM is gone it's closer to a pattern. The AMM was not in amm.js

Reply:
I changed it, but I felt hesitant the whole time I was doing it. 

The package is `auction`, and the thing you get from the `auctionKit` is an auction. The top-level of the contract is the part that knows how to be an auctioneer and manage the whole auction. But if the naming convention insists that the kit takes the name of the contract file, so be it.